build.xml

```diff
@@ -165,13 +167,26 @@
         </condition>
         <echo if:set="saxon-available" message="saxon available: ${saxon-available}"/>
         <echo unless:set="saxon-available" message="saxon available: false"/>
+        <!-- check if Stylesheets are available -->
+        <fileset id="version.file" dir="${dir.lib.stylesheets}">
+            <patternset>
+                <include name="VERSION"/>
+            </patternset>
+            <contains text="${stylesheets.version}" />
+        </fileset>
+        <condition property="stylesheets-available">
+            <resourcecount when="equal" count="1" refid="version.file"/>
+        </condition>
```

Comment:
```suggestion
        <condition property="stylesheets-available">
            <and>
                <available file="${dir.lib.stylesheets}"/>
                <resourcecount when="equal" count="1" refid="stylesheets.version.file"/>
            </and>
        </condition>
```

Reply:
Thanks for the suggested check for the VERSION file. It failed when running `ant reset` beforehand, since it does not have a stylesheet directory then. So finally we can bring back the conditional check 🎉 

I also tested with a different VERSION number in the file. So when you have 7.55 specified in your VERSION file, it now downloads the version requested by the ant script. 👍